modules/openapi-generator/src/main/resources/typescript-nestjs/api.module.mustache

```diff
@@ -0,0 +1,40 @@
+import { DynamicModule, HttpService, HttpModule, Module, Global } from '@nestjs/common';
+import { Configuration } from './configuration';
+import { BASE_PATH } from './variables';
+
+{{#apiInfo}}
+{{#apis}}
+import { {{classname}} } from './{{importPath}}';
+{{/apis}}
+{{/apiInfo}}
+
+@Global
+@Module({
+  imports:      [ HttpModule ],
```

Comment:
Here the HttpModule is used as a static module so yes it can be, but I actually should probably change it to a dynamic module with `HttpModule.register({})`